src/components/Dashboard/Dashboard.js

```diff
@@ -19,6 +48,34 @@ export const Dashboard = () => {
           <span className="arrow"></span>
         </div>
       </header>
+      <div className="dpsg-wrapper">
+        <div className="dpsg-content-wrap">
+          <div className="dp-container">
+            <S.Subtitle>
+              <span>
+                <img
+                  src="https://cdn.fromdoppler.com/doppler-ui-library/latest/img/asset-enterprise.svg"
+                  alt="icon-title"
+                />
+              </span>
+              Aprende con Doppler
+            </S.Subtitle>
+            <div className="dp-rowflex">
+              {carouselsList.map((carousels) => (
```

Comment:
I'm not sure about this, perhaps I want to use 2 orange carousels anywhere in the app, and then I will be forced to search for the color outside the carousel component. And cannot reuse the colors defined here. If you are sure of this approach I would add a constant in utils like this:
```
export const carouselColors = [orange: 'orange', purple:'purple']
```
Something like this:
```
<Carousel id="someId" color={carouselColors.orange}>
```
I also think it would be more declarative to have both carousel explicitly appart, since the source of the posts has nothing to do with one another. 
```
<Carousel id="fist" color={carouselColor.orange}>
                    {postList.map((post) => (
                      <Slide>
                        <TextPreviewPost post={post} />
                      </Slide>
                    ))}
</Carousel>
<Carousel id="second" color={carouselColor.purple}>
                    {postList.map((post) => (
                      <Slide>
                        <TextPreviewPost post={post} />
                      </Slide>
                    ))}
</Carousel>
```

Reply:
I thought of it that way so that the carousels can be created automatically and it is less code

Reply:
But is not real, because we will use this component in another pages, less code doesn`t mean better. We should reuse the colors somehow also.
Why would you, in a real scenario, render a list of carousels by colors? 🤔